packages/core/android/src/test/kotlin/com/datadog/reactnative/DdSdkTest.kt

```diff
@@ -1263,6 +1281,40 @@ internal class DdSdkTest {
         }
     }
 
+    @Test
+    fun `𝕄 send long tasks 𝕎 frame time is over threshold() {}`(
+        @LongForgery timestampNs: Long,
```

Comment:
By default `min = Long.MIN_VALUE`, so to cut off unrealistic timestamps let's specify the lower boundary.

```suggestion
        @LongForgery(min = 0L) timestampNs: Long,
```